packages/library/src/commonMain/kotlin/io/realm/BaseRealm.kt

```diff
@@ -66,7 +66,17 @@ public abstract class BaseRealm internal constructor(
         log.info("Realm opened: ${configuration.path}")
     }
 
-    fun <T : RealmObject> objects(clazz: KClass<T>): RealmResults<T> {
+    /**
+     * Returns the results of querying for all objects of a specific type.
+     *
+     * For a [Realm] instance this will reflect the state of the Realm at the invocation time, thus
```

Comment:
```suggestion
     * For a [Realm] instance this will reflect the state of the realm at the invocation time, thus
```
To avoid confusion between "Realm" (the product name shorthand), "MongoDB Realm" (the product name), "Realm Database" (the client-side database) and "realm" (an instance of a connection to a single file containing Realm Database data), we usually use lowercase "realm" in cases like this. I know it's a little confusing since Kotlin uses uppercase for the first letter of class names, so up to you if you'd like to follow this convention.

Reply:
```suggestion
     * For a [Realm] instance this reflects the state of the Realm at the invocation time, thus
```
Additionally, active voice can make this easier to read for end users.